schemas/openapi_v1.yml

```
@@ -428,6 +431,23 @@ components:
           $ref: "#/components/schemas/formatting"
         readOnly:
           type: boolean
+    timelineCell:
+      type: object
+      required:
+        - type
+        - id
+        - dataLinks
+      properties:
+        type:
+          $ref: "#/components/schemas/cellType"
+        id:
+          type: string
+        dataLinks:
+          type: array
+          items:
+            type: string
+        readOnly:
+          type: booleanØ
```

Think you dropped something :)

```suggestion
          type: boolean
```

---

i dont even know how i typed that 😅